crates/cargo-codspeed/src/run.rs

```diff
@@ -157,10 +158,16 @@ pub fn run_benches(
                 if status.success() {
                     Ok(())
                 } else {
-                    Err(anyhow!(
+                    let code = status
+                        .code()
+                        .or(status.signal().map(|s| 128 + s))
```

Comment:
Let's add a small comment saying that returning 128+n is a convention for "system stopped by signal n"